services/ui-src/src/libs/spaLib.ts

```diff
@@ -4,118 +4,252 @@ export interface SPAi {
-    state: "KS",
-    name: "Health Home Serving Serious Mental Illness (SMI)",
-  },
-  {
-    id: "20-0005",
-    state: "KS",
-    name: "Health Home Chronic Conditions - Asthma",
-  },
-  {
-    id: "13-012",
-    state: "ME",
-    name: "Maine Stage A Health Home Targeting Individuals with Chronic Conditions",
-  },
-  { id: "18-0002", state: "ME", name: "Maine Behavioral Health Home" },
-  {
-    id: "18-0032",
-    state: "ME",
-    name: "Maine Health Home for Beneficiaries Receiving Medication Assisted Treatment for Opioid Addiction",
-  },
-  { id: "20-0006", state: "MD", name: "Maryland Health Home Services" },
-  {
-    id: "16-1500",
-    state: "MI",
-    name: "Chronic Care Model for Individuals with Serious and Persistent Mental Health Conditions",
-  },
-  { id: "15-2000", state: "MI", name: "Michigan Care Team" },
-  { id: "18-1500", state: "MI", name: "Opioid Health Home" },
-  { id: "19-0015", state: "MN", name: "Minnesota Behavioral Health Homes" },
-  {
-    id: "19-0020",
-    state: "MO",
-    name: "Community Mental Health Center Health Home",
-  },
-  { id: "19-0003", state: "MO", name: "Primary Care Clinic Health Home" },
-  {
-    id: "16-0001",
-    state: "NJ",
-    name: "New Jersey Behavioral Health Home (Adults)",
-  },
-  {
-    id: "16-0002",
-    state: "NJ",
-    name: "New Jersey Behavioral Health Home (Children)",
-  },
-  { id: "18-0002", state: "NM", name: "CareLink New Mexico" },
-  {
-    id: "20-0054",
-    state: "NY",
-    name: "Health Home for High-Cost, High-Needs Enrollees ",
-  },
-  { id: "17-0025", state: "NY", name: "New York I/DD Health Home Services" },
-  { id: "14-0012", state: "OK", name: "Oklahoma Health Home (Adults)" },
-  { id: "14-0011", state: "OK", name: "Oklahoma Health Home (Children)" },
-  {
-    id: "18-009",
-    state: "RI",
-    name: "Comprehensive Evaluation, Diagnosis, Assessment, Referral, & Reevaluation (CEDARR) Family Centers Health Home",
-  },
-  {
-    id: "18-006",
-    state: "RI",
-    name: "Community Mental Health Organizations Health Home",
-  },
-  {
-    id: "16-006",
-    state: "RI",
-    name: "Rhode Island Opioid Treatment Program Health Home Services",
-  },
-  { id: "20-0008", state: "SD", name: "South Dakota Health Home" },
-  {
-    id: "16-004",
-    state: "TN",
-    name: "Tennessee Health Link Health Home Program ",
-  },
-  {
-    id: "14-007",
-    state: "VT",
-    name: "Vermont Health Home for Beneficiaries Receiving Medication Assisted Treatment for Opioid Addiction ",
-  },
-  { id: "20-0031", state: "WA", name: "Washington Health Home Services" },
-  {
-    id: "16-0007",
-    state: "WV",
-    name: "West Virginia Health Home for Individuals with Bipolar Disorder at Risk for Hepatitis Type B and C",
-  },
-  {
-    id: "16-0008",
-    state: "WV",
-    name: "West Virginia Health Home for Individuals with Chronic Conditions",
-  },
-  { id: "12-008", state: "WI", name: "Individuals with HIV/AIDS" },
-];
+export const SPA: any = {
```

Comment:
```suggestion
export const SPA: {[year: string]: SPAi[]} = {
```
In the interest of using the SPAi type, could we do this?